int32 order
bool use_with_local_frame
int32 knot_division
float64[] knot
geometry_msgs/Point[] global_control_points